test/modules/carto/layers/h3-tileset-2d.spec.ts

```diff
@@ -49,6 +49,45 @@ test('H3Tileset2D', async t => {
   t.end();
 });
 
+test('H3Tileset2D#tileSize', async t => {
+  const tileset512 = new H3Tileset2D({tileSize: 512});
+  const tileset1024 = new H3Tileset2D({tileSize: 1024});
+  const tileset2048 = new H3Tileset2D({tileSize: 2048});
+
+  const viewport = new WebMercatorViewport({
+    latitude: 0,
+    longitude: 0,
+    zoom: 6,
+    width: 300,
+    height: 200
+  });
+
+  const indicesSort = (a, b) => parseInt(a.i, 16) - parseInt(b.i, 16);
+  const indices512 = tileset512.getTileIndices({viewport}).sort(indicesSort);
+  const indices1024 = tileset1024.getTileIndices({viewport}).sort(indicesSort);
+  const indices2048 = tileset2048.getTileIndices({viewport}).sort(indicesSort);
+
+  t.deepEqual(
+    indices512,
+    [
+      {i: '8274effffffffff'},
+      {i: '827547fffffffff'},
+      {i: '82754ffffffffff'},
+      {i: '82755ffffffffff'},
+      {i: '82756ffffffffff'}
+    ],
+    'indices @ 512px'
+  );
+  t.deepEqual(indices1024, [{i: '81757ffffffffff'}], 'indices @ 1024px');
+  t.deepEqual(indices2048, [{i: '8075fffffffffff'}], 'indices @ 2048px');
+
+  t.equal(tileset512.getTileZoom(indices512[0]), 2, 'zoom @ 512px');
+  t.equal(tileset1024.getTileZoom(indices1024[0]), 1, 'zoom @ 1024px');
+  t.equal(tileset2048.getTileZoom(indices2048[0]), 0, 'zoom @ 2048px');
```

Comment:
I added this test because it's reassuring to see decreasing zooms with increasing file size, but, what do these zoom levels actually mean? 😓

Reply:
You should increase the viewport size so your tests produce more tiles. Then you should be able to observe that each doubling of the tile size reduces the number of tiles by roughly a factor of 4, as expected

Reply:
Great, thanks! They are indeed.

Reply:
If you use equation B in the above tests, do you also observe the reduction factor also to be 4? I would expect it to be 7, which is the number of children a h3 cell has